packages/typespec-ts/test/modularUnit/scenarios/operations/pathParam/optionalPath.md

```diff
@@ -28,7 +28,7 @@ export function _readSend(
   options: ReadOptionalParams = { requestOptions: {} }
 ): StreamableMethod {
   const path = expandUrlTemplate(
-    "/{/param}",
+    "{/param}",
```

Comment:
refer to : https://github.com/Azure/typespec-azure/pull/2605/files#diff-7418aca26c7c5543c790062b91bc3f5ee1a0e8355baf9857b5ee15895a47c656R20